Source/MOLAuthenticatingURLSession.m

```diff
@@ -60,33 +60,28 @@ - (void)setUserAgent:(NSString *)userAgent {
   self.sessionConfig.HTTPAdditionalHeaders = addlHeaders;
 }
 
-#pragma mark Server Roots properties
+#pragma mark Server Roots
 
 - (void)setServerRootsPemFile:(NSString *)serverRootsPemFile {
-  if (!serverRootsPemFile) {
-    _serverRootsPemFile = nil;
-    return;
-  }
+  if (!serverRootsPemFile) return;
   NSError *error;
   NSData *rootsData = [NSData dataWithContentsOfFile:serverRootsPemFile
                                              options:0
                                                error:&error];
   if (!rootsData) {
-    [self log:@"Unable to read server root certificate file %@ with error: %@",
-        self.serverRootsPemFile, error.localizedDescription];
+    return [self log:@"Unable to read server root certificate file %@ with error: %@",
+            serverRootsPemFile, error.localizedDescription];
   }
-  self.serverRootsPemData = rootsData;
+  [self setServerRootsPemData:rootsData];
 }
 
 - (void)setServerRootsPemData:(NSData *)serverRootsPemData {
```

Comment:
Need to add a check for serverRootsPemData being nil, which nils out self.anchors